packages/paste-website/src/pages/blog/2022-10-18-danger-js-helping-paste-focus-on-paste.mdx

```diff
@@ -0,0 +1,10 @@
+---
+title: Danger JS- Helping Paste Focus on Paste
```

Comment:
question: should there be a space after `JS` and before the hyphen `-`?

Reply:
yis, confirmed 
<img width="363" alt="image" src="https://user-images.githubusercontent.com/77117846/199123905-1b2c88ec-24a4-44f3-8a15-cb6e7dd8b953.png">

Reply:
No, no, _after_ the JS

`Danger JS- Helping Paste Focus on Paste`

to 

`Danger JS - Helping Paste Focus on Paste`